doc/techref/projections.md

```diff
@@ -0,0 +1,50 @@
+---
+myst:
+  substitutions:
+    lon0: "lon{sub}`0`",
+    lat0: "lat{sub}`0`",
+    lon1: "lon{sub}`1`",
+    lat1: "lat{sub}`1`",
+    lat2: "lat{sub}`2`",
+    lonp: "lon{sub}`p`",
+    latp: "lat{sub}`p`",
+    url_doc_proj: "https://www.pygmt.org/latest/projections/",
+---
+
+# GMT Map Projections
+
+The table below shows the projection codes for the 31 GMT projections:
+
+| PyGMT Projection Argument | Projection Name |
+| --- | --- |
+| **A**{{ lon0 }}/{{ lat0 }}[/*horizon*]/*width*              | {{ '[Lambert azimuthal equal area]({}azim/azim_equidistant.html)'.format(url_doc_proj) }} |
```

Comment:
```suggestion
| **A**{{ lon0 }}/{{ lat0 }}[/*horizon*]/*width*              | {doc}`Lambert azimuthal equal area </projections/azim/azim_equidistant>` |
```

Reply:
Updated for all rows in commit https://github.com/GenericMappingTools/pygmt/pull/3363/commits/30dbb5d0d5a06df985f9d8b659679ea3926cba08